Components/badge.css

```diff
@@ -0,0 +1,108 @@
+.badge{
+    background-color: var(--clr-secondary);
+    color: var(--text-clr-secondary);
+    padding: 0.25em;
+    text-align: center;
+    z-index: 1;
+    display: inline;
+}
+
+.badge--round{
+    border-radius: 50%;
+}
+
+.badge__status--online{
+    background-color: #22c55e;
+}
+
+.badge__status--busy{
+    background-color: #dc2626;
+}
+
+.badge--border-white{
+    border: 0.2em solid #FFFFFF;
+}
+
+.badge--small{
+    font-size: 1rem;
+    min-width: 1.6rem;
+    height: 1.6rem;
+    text-align: center;
+}
+
+.badge--xs{
+    font-size: 0.6rem;
```

Comment:
Can use pixels instead of rem when size is small.

Reply:
Ok thanks for the suggestion